doc/classes/SpinBox.xml

```diff
@@ -71,8 +71,98 @@
 		</member>
 	</members>
 	<theme_items>
+		<theme_item name="down_disabled_icon_modulate" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 0.5)">
+			Down button icon modulation color, when the button is disabled.
+		</theme_item>
+		<theme_item name="down_hover_icon_modulate" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Down button icon modulation color, when the button is hovered.
+		</theme_item>
+		<theme_item name="down_icon_modulate" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 1)">
+			Down button icon modulation color.
+		</theme_item>
+		<theme_item name="down_pressed_icon_modulate" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Down button icon modulation color, when the button is being pressed.
+		</theme_item>
+		<theme_item name="up_disabled_icon_modulate" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 0.5)">
+			Up button icon modulation color, when the button is disabled.
+		</theme_item>
+		<theme_item name="up_hover_icon_modulate" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Up button icon modulation color, when the button is hovered.
+		</theme_item>
+		<theme_item name="up_icon_modulate" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 1)">
+			Up button icon modulation color.
+		</theme_item>
+		<theme_item name="up_pressed_icon_modulate" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Up button icon modulation color, when the button is being pressed.
+		</theme_item>
+		<theme_item name="buttons_vertical_separation" data_type="constant" type="int" default="0">
+			Vertical separation between the up and down buttons.
+		</theme_item>
+		<theme_item name="buttons_width" data_type="constant" type="int" default="16">
+			Width of the up and down buttons. If smaller than any icon set on the buttons, the said icon may overlap neighboring elements, unless [theme_item set_min_buttons_width_from_icons] is enabled.
+		</theme_item>
+		<theme_item name="field_and_buttons_separation" data_type="constant" type="int" default="2">
+			Width of the horizontal separation between the text input field ([LineEdit]) and the buttons.
+		</theme_item>
+		<theme_item name="set_min_buttons_width_from_icons" data_type="constant" type="int" default="1">
```

Comment:
I have to say that that a property name prefixed with `set_` is rather unorthodox. I have no suggestions at the moment but it doesn't feel right.

Reply:
Would something like "use_icon_width_as_min_button_width" work out better?

Sure, "set" and "get" may seem odd here with a programmer mindset as this is not a proper setter/getter in OOP sense. More a contraction of "Set the minimum button width according to the icon width". I thought about it more from a user standpoint that may be not biased with programming terms.

Reply:
Yes, I personally find that name to be much more suitable. Suggestions are still welcome if there's a briefer name.